docs/source/usage/plugins/openPMD.rst

```diff
@@ -43,7 +43,7 @@ The openPMD API will parse the file name to decide the chosen backend and iterat
 In order to set defaults for these value, two further options control the filename:
 
 * ``--openPMD.ext`` sets the filename extension.
-  Possible extensions include ``bp`` for the ADIOS backends (default), ``h5`` for HDF5 and ``sst`` for Streaming via ADIOS2/SST.
+  Possible extensions include ``bp`` for the ADIOS2 backends (default), ``h5`` for HDF5 and ``sst`` for Streaming via ADIOS2/SST.
```

Comment:
Is it not possible to write ADIOS(1) output anymore? I thought openPMD-api supports both ADIOS and ADIOS2.

Reply:
I also thought openPMD API would still use ADIOS1 if that's how it was compiled. Or if both were found, i think there is an environment variable that controlls which one to use for `.bp`

Reply:
Yes it is still possible. I changed it because we should not recommend using ADIOS1 anymore.
In openPMD ADIOS1 and ADIOS2 are mutually exclusive as I know e.g. if you set in PIConGPU the file extension to `bp` it will use ADIOS2 if possible.